bloom/commands/git/import_upstream.py

```diff
@@ -38,6 +38,7 @@
 import tarfile
 import tempfile
 
+from io import open
```

Comment:
Doing `import io` and using `io.open()` everywhere is probably slightly better as to not overwrite a built-in, but I personally don't mind either way.